data/ka'het/ka'het outfits.txt

```diff
@@ -10,6 +10,100 @@
 
 # Weapons:
 
+outfit "Ka'het Nullifier"
+	category "Guns"
+	cost 3062000
+	thumbnail "outfit/ka'het nullifier"
+	"mass" 68
+	"outfit space" -97
+	"weapon capacity" -97
+	"gun ports" -1
+	weapon
+		sprite "projectile/ionball"
+			"frame rate" 15
+			"rewind"
+		sound "ionball"
+		turn 73
+		"fire effect" "finisher sparks" 20
+		"hit effect" "ionball ring" 15
+		"hit effect" "ionball impact"
+		"hit effect" "ionball fragment" 24
+		"live effect" "ionball spark" 140
+		"live effect" "ionball trail" 140
+		"live effect" "ionball twinkle" 560
+		"velocity" 9
+		"lifetime" 200
+		"reload" 350
+		"firing energy" 710
+		"firing force" 90
+		"firing heat" 160
+		"hit force" 50
+		"shield damage" 120
+		"hull damage" 40
+		"ion damage" 150
+		"inaccuracy" 0.2
+		"blast radius" 20
+	description "The Nullifier was one of the largest weapons the Builders ever developed, created specifically to be carried in battle by the Vareti. Although it does basically no damage to shields and hull, one shot is capable of disabling most kinds of reactors in a matter of seconds."
```

Comment:
```suggestion
	description "The Nullifier was one of the largest weapons the Builders ever developed, created specifically to be carried in battle by the Vareti. Although it does almost no damage to shields and hull, one shot is capable of disabling most kinds of reactors in a matter of seconds."
```
OR
```suggestion
	description "The Nullifier was one of the largest weapons the Builders ever developed, created specifically to be carried in battle by the Vareti. Although it does virtually no damage to shields and hull, one shot is capable of disabling most kinds of reactors in a matter of seconds."
```
This is just personal preference. Just not a big fan of `basically`, but it's up to you.